src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/FullScreen/FullScreenEffectRouter.shared.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class FullScreenEffectRouter : RoutingEffect
+	{
+		public FullScreenEffectRouter()
+			: base(EffectIds.FullScreenEffect)
+		{
+#if __ANDROID__
+			if (System.DateTime.Now.Ticks < 0)
+				_ = new Xamarin.CommunityToolkit.Android.Effects.FullScreenEffectRouter();
+#else
+			throw new NotImplementedException();
```

Comment:
not sure if we want to throw an exception here...

Reply:
This is temporary because this pr is implementing Android part only.